docs/cpp/configure-intellisense.md

```diff
@@ -144,49 +150,52 @@ Select a tab to see some samples of a `c_cpp_configuration.json` file based on y
+* Additional include paths, such as references to one or multiple different libraries
+* Specific compiler arguments that influence the behavior of the language(and therefore IntelliSense)
 
-- additional include paths, such as references to one or multiple different libraries
-- specific compiler arguments that influence the behavior of the language(and therefore IntelliSense)
+There are multiple other ways to configure IntelliSense. You can provide these additional configurations either through:
 
-There are multiple other ways to configure IntelliSense. You can provide these additional configurations either through the `c_cpp_properties.json` file and related settings, a custom configuration provider in the form of another VS Code extension (for example, the [Makefile Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.makefile-tools) or [CMake Tools](/docs/cpp/CMake-linux.md) extension), or a `compile_commands.json` file.
+* The `c_cpp_properties.json` file and related settings
+* A custom configuration provider in the form of another VS Code extension (for example, the [Makefile Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.makefile-tools) or [CMake Tools](/docs/cpp/cmake-linux.md) extensions)
+* A `compile_commands.json` file
 
-### Configuration Providers
+### Configuration providers
 
-A custom configuration provider is another extension in VS Code which can potentially provide more accurate C++ IntelliSense configuration than the C/C++ Extension. For example, for the CMake or Make build systems, where the [Makefile Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.makefile-tools) or [CMake Tools](/docs/cpp/CMake-linux.md) extensions can be configuration providers. To add an extension as a configuration provider, either select the extension through the configuration quick pick, add it to configuration UI by editing the **Configuration provider** field under **Advanced Settings**, or add the `configurationProvider` field to your `c_cpp_properties.json` file. For example, for the CMake extension, the path to add would be `ms-vscode.cmake-tools`.
+A custom configuration provider is another extension in VS Code that can potentially provide more accurate C++ IntelliSense configuration than the C/C++ extension. For example, for the CMake or Make build systems, where the [Makefile Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.makefile-tools) or [CMake Tools](/docs/cpp/cmake-linux.md) extensions can be configuration providers. To add an extension as a configuration provider, either select the extension through the configuration Quick Pick, add it to configuration UI by editing the **Configuration provider** field under **Advanced Settings**, or add the `configurationProvider` field to your `c_cpp_properties.json` file. For example, for the CMake extension, the path to add would be `ms-vscode.cmake-tools`.
 
```

Comment:
Please add this image: 
![image](https://github.com/microsoft/vscode-docs/assets/67650598/7922cb6d-d2df-44d7-9f61-fc84ff562b5f)